fendermint/actors/gas_market/src/lib.rs

```diff
@@ -118,15 +135,15 @@ impl ActorCode for EIP1559GasMarketActor {
 
     actor_dispatch! {
         Constructor => constructor,
-        SetBlockGasLimit => set_block_gas_limit,
-        GetState => get_state,
-        UpdateBlockGasConsumption => update_block_gas_consumption,
+        SetConstants => set_constants,
+        CurrentGasReading => current_gas_reading,
```

Comment:
We know this is about gas, we don't need to be pedantic :-)

```suggestion
        CurrentReading => current_reading,
```